pages/chat/_entity.vue

```diff
@@ -31,11 +30,10 @@
       <div class="d-flex flex-column flex-grow-1">
         <!-- Message List -->
         <simplebar
-          class="simplebar wide-scrollbar flex-grow-1" 
-          data-simplebar-auto-hide="false" 
+          class="simplebar simplebar-no-gutter wide-scrollbar flex-grow-1" 
+          data-simplebar-auto-hide="true"
```

Comment:
Why does the message window (attempt to) auto hide?
```suggestion
          data-simplebar-auto-hide="false"
```